### PR TITLE
Loader and fixes

### LINKED ISSUE #98: Add page loader
We need to have a loader, prefferably a picture/gif to show if something is loading in the background.

### PR DESCRIPTION
**Pull request by wangtiger317**

### CODE REVIEW

**Review by ethedev — CHANGES_REQUESTED** [review]
@wangtiger317 the loader must only be shown in case it was needed on actual data load or very necessary charting load, not everytime on each single mouse click

**Comment by ethedev** [comment]
**Notes:**
- [ ] dropdowns on both Help and Dates (in markets page) are not closing after you click outside, it should be
- [ ] the loader isnt working how its supposed to

**Comment by wangtiger317** [comment]
> **Notes:**
> 
> * [ ]  dropdowns on both Help and Dates (in markets page) are not closing after you click outside, it should be
> * [ ]  the loader isnt working how its supposed to

if so, I am not sure for loader well. at the moment I will work on the dropdown of market page

**Comment by ethedev** [comment]
> @wangtiger317 the loader must only be shown in case it was needed on actual data load or very necessary charting load, not everytime on each single mouse click

**Comment by wangtiger317** [comment]
> > @wangtiger317 the loader must only be shown in case it was needed on actual data load or very necessary charting load, not everytime on each single mouse click

ok. will try on that side

File: src/components/SecretImage.vue [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<template>
```
Comment (ethedev) [inline src/components/SecretImage.vue]: rename to `LoaderImage`

File: src/views/ExploreSynths.vue [hunk 2]
```diff
@@ -259,7 +263,8 @@ export default {
   },
   methods: {
     filter(event) {
-      this.filter_string = event.target.value
+      this.filter_string = event.target.value;
+      console.log(this.synths)
```
Comment (ethedev) [inline src/views/ExploreSynths.vue]: remove console log after you finish using it
```suggestion
```

**Review by ethedev — CHANGES_REQUESTED** [review]

File: src/views/ExploreSynths.vue (hunk 2 above)
Comment (wangtiger317) [reply src/views/ExploreSynths.vue]: okay

File: src/components/SecretImage.vue (hunk 1 above)
Comment (wangtiger317) [reply src/components/SecretImage.vue]: ok

**Review by ethedev — APPROVED** [review]

**Pull request merged**